tests/e2e/core/pg/10-create-cluster.yaml

```diff
@@ -14,6 +14,26 @@ spec:
     resources:
       cpu: "1"
       memory: 1G
+    affinity:
+      nodeAffinity:
+        preferredDuringSchedulingIgnoredDuringExecution:
+        - weight: 5
+          preference: 
+            matchExpressions:
+            - key: another-node-label-key
+              operator: In
+              values:
+              - another-node-label-value
```

Comment:
I'm commenting here but this is a generic comment for all tests added in this PR.
Even though we are setting this node affinity as preferred which shouldn't impact the scheduling of the pods, I'd still move these tests to the `tests/integration/features` directory because we don't need to complicate the core e2e tests with this. Since we only want to assert that the everest operator correctly translated the affinity settings to the pxc/psmdb/pg CRs, we don't need the pxc/psmdb/pg operators to be running, IMO the affinity tests should be run as an integration test and we'd avoid polluting the core e2e tests with some affinity settings that can interfere with the results.